sdk/core/core-auth/src/tokenCredential.ts

```diff
@@ -47,16 +47,18 @@ export interface GetTokenOptions {
      */
     tracingContext?: TracingContext;
   };
-
-  /**
-   * Allows specifying a tenantId. Useful to handle challenges that provide tenant Id hints.
-   */
-  tenantId?: string;
-
   /**
    * Claim details to perform the Continuous Access Evaluation authentication flow
    */
   claims?: string;
+  /**
+   * Indicates whether to enable the Continuous Access Evaluation authentication flow
+   */
+  enableCAE?: boolean;
```

Comment:
Is this name appropriate or should it be `enlableCae` since it's a three letter acronym versus just a two or more.

Reply:
I'll change it to `enableCae`

Reply:
done